x-pack/plugins/apm/common/rules/apm_rule_types.ts

```diff
@@ -40,6 +40,15 @@ export enum AggregationType {
   P99 = '99th',
 }
 
+export const BUCKET_SIZE = 5;
```

Comment:
Again, I don't think we want to declare this as a constant at this level. It's merely an implementation detail for the preview charts.

Reply:
removed `BUCKET_SIZE` from `common`